tests/decap/test_decap.py

```diff
@@ -46,6 +46,30 @@
 ]
 
 
+@pytest.fixture(autouse=True)
+def ignore_expected_loganalyzer_exceptions(duthosts, loganalyzer):
+    """
+        Ignore expected failures logs during test execution.
+
+        We disable LLDP during the test, so we expect to see "lldp not running"
+        messages in the logs. All other errors should be treated as errors.
+
+        Args:
+            duthost: DUT fixture
+            loganalyzer: Loganalyzer utility fixture
+    """
+    ignoreRegex = [
```

Comment:
Can we move these ignore list in a file?

Reply:
It would ignore with if condition, so I tend not to put it in a file